Content.Server/Atmos/Monitor/Systems/AirAlarmSystem.cs

```diff
@@ -219,14 +222,23 @@ private void OnClose(EntityUid uid, AirAlarmComponent component, BoundUIClosedEv
             RemoveActiveInterface(uid);
     }
 
+    private void OnInit(EntityUid uid, AirAlarmComponent comp, ComponentInit args)
+    {
+        _deviceLink.EnsureSourcePorts(uid, comp.DangerPort, comp.WarningPort, comp.NormalPort);
+
+        // for mapped linked air alarms, start with high so when it changes for the first time it goes from high to low
+        // without this the output would suddenly get sent a low signal after nothing which is bad
+        _deviceLink.SendSignal(uid, GetPort(comp), true);
+    }
+
     private void OnShutdown(EntityUid uid, AirAlarmComponent component, ComponentShutdown args)
     {
         _activeUserInterfaces.Remove(uid);
     }
 
     private void OnInteract(EntityUid uid, AirAlarmComponent component, InteractHandEvent args)
     {
-        if (!_interactionSystem.InRangeUnobstructed(args.User, args.Target))
+        if (!_interaction.InRangeUnobstructed(args.User, args.Target))
             return;
```

Comment:
This check should be unneeded as SharedInteractionSystem checks inRangeUnobstructed for this.